src/copy.c

```diff
@@ -211,15 +211,15 @@ timescaledb_CopyFrom(CopyState cstate, Relation main_rel, List *range_table, Hyp
 	errcallback.previous = error_context_stack;
 	error_context_stack = &errcallback;
 
+    ChunkInsertState *prev_cis = NULL;
```

Comment:
We try to keep all declaration at the beginning of blocks as per C99. Can you move the up to line ~85